django_snow/migrations/0003_auto_20190226_0119.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 2.0.13 on 2019-02-26 01:19
+
+import django.core.validators
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('django_snow', '0002_changemgmt_add_createtime_closetime'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
```

Comment:
Looks like this is creating new table and deleting the existing one instead of renaming the table. These might help in generating the migrations for renaming the model:
* https://docs.djangoproject.com/en/2.1/ref/migration-operations/#renamemodel
* https://stackoverflow.com/questions/25091130/django-migration-strategy-for-renaming-a-model-and-relationship-fields